api/go.mod

```diff
@@ -237,6 +233,7 @@ replace (
 	k8s.io/controller-manager => k8s.io/controller-manager v0.22.7
 	k8s.io/cri-api => k8s.io/cri-api v0.22.7
 	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.22.7
+	k8s.io/klog/v2 => k8s.io/klog/v2 v2.9.0
```

Comment:
Where is this used?

Reply:
It's used as an indirect dependency in [mlp]( https://github.com/gojek/mlp/blob/05ede139570e1a9fc271fb03dca3fcfe7deb6074/go.mod#L113).